packages/manager/apps/sign-up/src/index.js

```diff
@@ -62,6 +62,9 @@ angular
     TranslateServiceProvider.setUserLocale(userLocale);
     $translateProvider.use(userLocale);
   })
+  .config(/* @ngInject */ (ssoAuthenticationProvider) => {
+    ssoAuthenticationProvider.allowIncompleteNic(true);
```

Comment:
Is the `Public Cloud` application should also enable incomplete nic?

Reply:
As nobody knows exactly what can happen with incomplete nic, all applications do not allow incomplete nic before somebody detect that there is no risk to allow incomplete nics :-)

Reply:
It should be the other way round therefore `allowIncompleteNic(false)`

Reply:
Not for sign-up. The /auth page will create incomplete nics, then the creation script will redirect the incomplete nic to the sign-up app page. The app have to allow `incomplete` nics in order to complete them.

Reply:
Oopsie I misread, my bad